shiny/express/_mock_session.py

```diff
@@ -39,12 +58,81 @@ def on_ended(
     ) -> Callable[[], None]:
         return lambda: None
 
-    def make_scope(self, id: Id) -> Session:
+    def make_scope(self, id: Id) -> SessionProxy:
```

Comment:
Shouldn't this still be `-> Session`? I think of the fact that it's actually a `SessionProxy` as an implementation detail.